libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewContext.kt

```diff
@@ -13,7 +15,11 @@ internal class DropInNavigationViewContext(
     val lifecycleOwner: LifecycleOwner,
     val viewModel: DropInNavigationViewModel,
     val mapView: MapView,
-    val viewGroup: ViewGroup,
```

Comment:
it's a feature https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

"It makes version-control diffs cleaner" :)